pages/registration/requirements.md

```diff
@@ -115,7 +115,7 @@ Federal court .gov domains are linked to [uscourts.gov](https://uscourts.gov) an
 
 ### Native sovereign nations domains
 
-Native Sovereign Nations domains are coordinated with the following Chief Information Officer representative:
+Native Sovereign Nations domains are [coordinated](https://www.bia.gov/sites/bia.gov/files/assets/public/press_release/pdf/idc013756.pdf) with the following Chief Information Officer representative:
```

Comment:
It might be a good idea to cache a copy of this PDF and stuff it into the repo, even if we still link to the bia.gov version.